MonkeyWrench.Web.WebService/Authentication.cs

```diff
@@ -127,13 +127,24 @@ private static void VerifyAnonymousAllowed()
 
 		public static void VerifyUserInRole (HttpContext Context, DB db, WebServiceLogin login, string role, bool @readonly)
 		{
+			VerifyUserInRoles(Context, db, login, new string[] { role }, @readonly);
+		}
+
+		public static void VerifyUserInRoles (HttpContext Context, DB db, WebServiceLogin login, string[] roles, bool @readonly)
+		{
 			WebServiceResponse dummy = new WebServiceResponse ();
 			Authenticate (Context, db, login, dummy, @readonly);
 
-			if (!dummy.IsInRole (role)) {
-				log.InfoFormat ("The user '{0}' has the roles '{1}', and requested role is: {2}", login.User, dummy.UserRoles == null ? "<null>" : string.Join (",", dummy.UserRoles), role);
-				throw new UnauthorizedException ("You don't have the required permissions.");
+			foreach (var role in roles) {
```

Comment:
What happens here if one of the roles is an empty string?

I think that can happen if someone just puts in a comma for the field when editing the lane.